src/app/app.routes.ts

```diff
@@ -1,6 +1,6 @@
 import { Routes } from '@angular/router';
 import { LayoutComponent } from './components/layout/layout.component';
-import { Step1Component } from './components/step-1/step-1.component';
```

Comment:
I think it is better if you type out the name of the numbers like step-one instead of step-1

Reply:
OK